### DISCUSSION TITLE
Call opc ua method

### DISCUSSION THREAD

**Discussion by apost9lis** [frangoteam/FUXA#863] [Q&A] [answered]
Hello,
is opc ua method call supported from FUXA?

Thank you!

**Comment by unocelli** [comment, marked as answer]
Hi, is currently not supported.

**Reply by apost9lis** [reply]
Thank you!

**Comment by apost9lis** [comment]
Hi,
I have two question:
1.  Is it possible to use a script to call a method?
2.  Is there any way to execute a program written in C or a python script from the dashboard and use this to call the method?
I would like to apologize in advance for my ignorance. I am new in this.

**Reply by unocelli** [reply]
Hi,
you can configure that from the frontend (click event) you run a script on the server and the script runs a c or python application
